### ISSUE TITLE
Segfault when importing profile

### ISSUE THREAD

**Issue by cybiere** [krobelus/cyanide#23]
I copied my .tox profile (generated with qtox on my laptop) in .config/tox folder, and when I try to switch to it cyanide closes, and won't start again as it tries to load it at start.

Launched from cli, I have : 

```
[D] Cyanide::tox_thread:404 - tox_save_file "/home/nemo/.config/tox/me.tox"
[D] file_raw:41 - Read 33277 bytes ( /home/nemo/.config/tox/me.tox )
Segmentation Fault
```

Sad me is sad :(

**Comment by krobelus** [comment]
Fixed for unencrypted profiles as of bbc02ab69acd24a5e51c77841535310095bef682
I will try to add support for encrypted saves too now (if you set a password in qtox the tox save file and the history is encrypted nowadays)

**Comment by krobelus** [comment]
I finally got around to make it work properly...
It should work in the latest version
